tools/autograd/derivatives.yaml

```diff
@@ -1348,19 +1348,19 @@
 
 - name: sub.Tensor(Tensor self, Tensor other, *, Scalar alpha=1) -> Tensor
   self: handle_r_to_c(self.scalar_type(), grad)
-  other: handle_r_to_c(other.scalar_type(), -grad * alpha.conj())
+  other: handle_r_to_c(other.scalar_type(), maybe_multiply(-grad, alpha.conj()))
   result: self_t - maybe_multiply(other_t, alpha)
 
 - name: sub.Scalar(Tensor self, Scalar other, Scalar alpha=1) -> Tensor
   self: handle_r_to_c(self.scalar_type(), grad)
   result: auto_element_wise
 
 - name: rsub.Tensor(Tensor self, Tensor other, *, Scalar alpha=1) -> Tensor
-  self: handle_r_to_c(other.scalar_type(), -grad * alpha.conj())
-  other: handle_r_to_c(self.scalar_type(), grad)
+  self: handle_r_to_c(self.scalar_type(), maybe_multiply(-grad, alpha.conj()))
```

Comment:
Would you elaborate on this change? Is there a test this can add that would have failed without it and now passes?

Reply:
I reckon that if you do rsub between a float and a complex tensor, this test should fail. otoh, we did not generate such samples in the OpInfo. Now, this kind of samples are generated in the next PR of this stack.